src/components/prompt-dialog-box.tsx

```diff
@@ -42,12 +50,24 @@ export default function PromptDialogBox({ children }: PromptDialogBoxProps) {
               Please enter the contract address of the stablecoin you want to use.
             </AlertDialogDescription>
           </AlertDialogHeader>
+          <Select onValueChange={(value) => setChainId(value)}>
```

Comment:
I think the information about chainId and their names is duplicated in a few places. I would suggest creating a separate file with an array of chainId and chainName pairs, and importing this file/array in all the places where it is needed. Then when we need to add more chains, we just need to include more chains in one place.

Reply:
Sure, I have made a separate file for storing chainId and chainName pairs.